### PR TITLE
Add translation to Arabic

### PR DESCRIPTION
**Pull request by celyes**
Added Arabic language translation to README.md

The translation can be found in README.ar.md

### CODE REVIEW

**Comment by nschonni** [comment]
If you want the UI portion translated as well, I think you need to add a new JSON translation file as well in https://github.com/giscus/giscus/tree/main/locales

**Comment by celyes** [comment]
Absolutely. I'm working on them. Thanks for reminding me 🙏

**Comment by nschonni** [comment]
Is there anything else that needs to change to support RTL languages?

**Comment by laymonage** [comment]
@nschonni I assume we'll need to add CSS rules to mirror some stuff (`text-align`, etc.)

**Comment by nschonni** [comment]
Not sure if something as simple as https://rtlcss.com/ would work, but probably doesn't need to happen in this PR

**Comment by celyes** [comment]
@nschonni finished translating both files. Would you take a look?

File: locales/ar/config.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  "theme=dark_dimmed": "GitHub Dark Dimmed",
+  "theme=dark_high_contrast": "GitHub Dark High Contrast",
+  "theme=dark_protanopia": "GitHub Dark Colorblind",
+  "theme=light_protanopia": "GitHub Light Colorblind",
+  "theme=transparent_dark": "داكن شفاف",
+  "theme=preferred_color_scheme": "نمط الألوان المفضل",
+  "theme=custom": "Custom (تجريبي)",
+
+  "enableGiscus": "تمكين giscus",
+  "addTheFollowingScriptTag": "أضف الكود الآتي <code><script></code> tag to your website's template where you want the comments to appear. If an element with the class <code>giscus</code> exists, the comments will be placed there instead.",
```
Comment (laymonage) [inline locales/ar/config.json:69]: Could you translate the rest of the string?

**Review by laymonage — COMMENTED** [review]
Thanks! I haven't got the time to work on proper RTL support (text alignment, etc.). Would you rather wait, or would it be okay to merge this without it?

File: locales/ar/config.json (hunk 1 above)
Comment (celyes) [reply locales/ar/config.json:69]: Sorry for the late  reply. 
The merge can wait until a proper RTL support is ready. For the translation, There are some technical words that if translated literally to Arabic, they would deviate the meaning, thus I chose to keep the meaning over doing this.

**Comment by celyes** [comment]
@nschonni would you approve ?

**Comment by nschonni** [comment]
@celyes sorry, I'm not a maintainer on this repo, so I can't approve anything for you

**Comment by laymonage** [comment]
Hey @celyes, sorry it took such a long time. I can get this merged soon if you don't want to wait for proper RTL support. Would really appreciate it if you could rebase this PR and translate some new strings I added since this PR was made.

Proper RTL support should probably be easier to implement once #354 is merged as [Tailwind v3 introduced RTL/LTR modifiers](https://tailwindcss.com/blog/tailwindcss-v3#rtl-and-ltr-modifiers). However, it's going to take time for me to implement the layout changes and make sure everything works properly.

**Comment by laymonage** [comment]
Superseded by #622. Feel free to open a new PR for improvements. Thanks!

**Pull request closed without merging**